randovania/games/fusion/logic_database/Sector 1 SRX.txt

```diff
@@ -1095,11 +1132,43 @@ Extra - room_id: [30]
       All of the following:
           Morph Ball
           Any of the following:
+              # Intended path
               Diffusion Missile Data and Missiles
-              # Diagonal Missile shot at the third Missile Block when standing close to the wall can hit it (may not even have to freeze the enemy)
-              Missiles ≥ 3 and Knowledge (Beginner) and Stand On Frozen Enemies (Beginner) and Can Freeze Enemies
+              All of the following:
+                  # Diagonal Missile shot at the third Missile Block when being close to the wall can hit it
+                  Missiles ≥ 3 and Knowledge (Advanced)
+                  # Freeze Ripper and stand on it:
+                  Stand On Frozen Enemies (Beginner) or Can Freeze Enemies
+                  # Ledge grab, and diagonally aim down after a neutral jump: https://youtu.be/8tUjNmvbRDE
+                  Movement (Intermediate)
+                  # Same but with Space Jump which gives you the guarantee of not falling down
+                  Space Jump
   > Door to Ripper Maze Access
-      Charge Beam or Missiles ≥ 10 or Screw Attack or Combat (Intermediate) or Can Freeze Enemies
+      Any of the following:
+          Charge Beam or Missiles ≥ 10 or Screw Attack or Combat (Intermediate) or Can Freeze Enemies
+          Any of the following:
+              # Get past the Rippers in Shaft
+              Screw Attack or Can Freeze Enemies
+              # Move to off the pillar, right as the top most Ripper turns to the right: https://youtu.be/66mAKLiF8z4
+              Combat (Intermediate)
+              # Damage boost
+              Damage Boosts (Beginner) and Normal Damage ≥ 50
+          Any of the following:
+              # Get past the Gerubus
+              Screw Attack
+              Missiles ≥ 15 and 10 DMG Missiles
+              Missiles ≥ 5 and 45 DMG Missiles
+              # Dodge: https://youtu.be/mpN1LGf4lmU
+              Morph Ball and Combat (Beginner)
+              # Damage Boost
+              Damage Boosts (Beginner) and Normal Damage ≥ 200
+          Any of the following:
+              # Get past the Rippers in corridor
+              Morph Ball or Screw Attack
+              # Freeze and do spinjump to glitch through it: https://youtu.be/6TSAshOOI-Q
+              Knowledge (Advanced) and Can Freeze Enemies
+              # Damage boost
+              Damage Boosts (Beginner) and Normal Damage ≥ 50
```

Comment:
I think this block might also need a requirement for getting out of this last climbing out w/wo hijump here?
that last jump off the ripper in the video seems like it might need `stand on frozen enemies or wall jumps or climb hijump wall`

Reply:
It doesnt, you can just normal powergrip this